ksql-rest-app/src/test/java/io/confluent/ksql/rest/entity/TableRowsEntityFactoryTest.java

```diff
@@ -100,6 +110,90 @@ public void shouldAddWindowedRowToValues() {
     assertThat(output.get(1), contains("y", now.toEpochMilli(), now.toEpochMilli(), false));
   }
 
+  @Test
+  public void shouldSupportNullStringColumn() {
```

Comment:
I think having all these tests is a little overkill :) i'd be happy with just one test that adds null in each row:
```
    newColumns.add(null);
    newColumns.add(null);
    newColumns.add(null);
    newColumns.add(null);
```